pkg/cli/mirror/options.go

```diff
@@ -0,0 +1,44 @@
+package mirror
+
+import (
+	"github.com/RedHatGov/bundle/pkg/cli"
+	imagemanifest "github.com/openshift/oc/pkg/cli/image/manifest"
+	"github.com/spf13/pflag"
+)
+
+type MirrorOptions struct {
+	*cli.RootOptions
+	OutputDir        string
+	ConfigPath       string
+	SkipImagePin     bool
+	ManifestsOnly    bool
+	From             string
+	ToMirror         string
+	DryRun           bool
+	SourceSkipTLS    bool
+	DestSkipTLS      bool
+	SkipVerification bool
+	SkipCleanup      bool
+	FilterOptions    imagemanifest.FilterOptions
+
+	BuildxPlatforms []string
+}
+
+func (o *MirrorOptions) BindFlags(fs *pflag.FlagSet) {
+	fs.StringVarP(&o.ConfigPath, "config", "c", o.ConfigPath, "Path to imageset configuration file")
+	fs.BoolVar(&o.SkipImagePin, "skip-image-pin", o.SkipImagePin, "Do not replace image tags with digest pins in operator catalogs")
+	fs.StringVar(&o.From, "from", o.From, "The archive file path.")
```

Comment:
Did we decide to improve the help text for this flag?

Reply:
@estroz We did on a previous PR. Was actually looking for ideas on exactly what we want here. I still agree that it needs to be updated. I can try to come up with something and update the PR.